python/pyspark/tests/test_readwrite.py

```diff
@@ -231,9 +231,10 @@ def test_newhadoop(self):
         )
         self.assertEqual(result, data)
 
-        fmt = "org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat"
```

Comment:
this is technically a followup of https://github.com/apache/spark/pull/35126 but just decided to piggy-back here.